README.md

```diff
@@ -1,2 +1,35 @@
 # DockTweet
-Docker command line interface to get tweets from a certain account. Saved in a .csv file
+Docker app is to get tweets from a certain account. Saved in a .csv file.
+
+## [Dockerhub](https://hub.docker.com/r/antero/docktweet/)
+
+<p align="center">
+  <img src="https://media.giphy.com/media/xThuVYHFQbZvAQU22Y/giphy.gif" width="350"/>
+</p>
+
+# Instructions 
+
+# 1 Create twitter app
+To use this docker you need to create an app on [Twitter](https://apps.twitter.com/) to provide app key
+
+# 2 Build docker 
```

Comment:
*Build the container*

ps.: This is not really necessary just to use the image... Remember that you have already uploaded one... So people can run using the uploaded image on dockerhub. `docker run antero/docktweet`. This is only necessary if they are going to make changes in the code.

I suggest you to break your README in two sections: Running and Contributing